projects/batfish-client/src/org/batfish/client/Client.java

```diff
@@ -315,6 +323,151 @@ private boolean answer(String questionTemplateName, String paramsLine,
       return execute(wItemAs, outWriter);
    }
 
+   /**
+    * Validate the json-encoded {@code value} matches the valid setting
+    *  specified by {@code variable}.
+    *  <p>If {@code value} does not match the valid setting,
+    *  {@link BatfishException BatfishException} will be thrown.</p>
+    */
+   static void validate(JsonNode value, Variable variable)
+         throws BatfishException {
+      if (variable.getType().getStringType() &&
+            !variable.getType().equals(Variable.Type.JSON_PATH)) {
+         if (!value.isTextual()) {
+            throw new BatfishException("The parameter value type: " + value
+                  .getNodeType() + " doesn't match the type required for " +
+                  "variable type: " + variable.getType());
+         }
+      }
+      switch (variable.getType()) {
+         case BOOLEAN:
+            if (!value.isBoolean()) {
+               throw new BatfishException("Expected a boolean for " +
+                     "variable type: " + variable.getType());
+            }
+            break;
+         case COMPARATOR:
+            String textContent = value.textValue();
+            if (!(textContent.equals(">") || textContent.equals(">=") ||
```

Comment:
Create a constant set of the valid comparators (e.g., `private static final Set<String> COMPARATORS = ...`) and then simply use `if (!COMPARATORS.contains(value.textValue()))`.